book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
-* Conclusions comparing risk groups. For example, thresholding the risks at $1.0$ means that $i$ and $j$ are in a low-risk group whilst $k$ is in a high-risk group.
+* The corresponding ranks for $i,j,k$ are $2,3,1$;
+* $k$ is at the least risk and $j$ is at the highest risk;
+* The risk of $i$ is slightly higher than that of $k$ but $j$'s risk is considerably higher than both the others.
+
+2) Conclusions comparing risk groups:
+
+* Thresholding risks at $0.4$ means $k$ is at a low-risk but $i$ and $j$ are high-risk.
+* Thresholding risks at $1.0$ means $i$ and $k$ are low-risk but $j$ is high-risk.
 
 Whilst many important conclusions can be drawn from these predictions, the values themselves have no meaning when not compared to other individuals.
-Interpretation of these rankings depends on the model class (for example, PH and AFT models have opposite interpretations, @sec-models-classical) and its parametrization or implementation in specific software.
-For some higher ranking implies higher risk whereas others may assume that higher ranking implies lower risk.
-In this book, a higher ranking will always imply a higher risk of event (as in the example above).
+Similarly, the values have no meanings across research, even if observation $k$ is at low-risk according to this sample, it may be high risk compared to another.
+
+### Distributions and risks
+
+In general it is not possible to easily convert a risk prediction to a distribution.
+However, this may be possible if the risk prediction corresponds to a particular model form.
+This is discussed further in @sec-survtsk-PI.
 
-Predicting rankings is the primary form of the *survival ranking task*, defined by predicting a continuous value, $g: \calX \rightarrow \calR$ where $\calR \subseteq \Reals$.
+More common is transformation from a distribution to a risk.
+Theoretically the simplest way to do so would be to take the mean or median of the distribution, however as will be discussed in detail in @sec-survtsk-time this is practically difficult.
+Instead, a stable approach is to use the 'ensemble' or 'expected mortality' to create a measure of risk [@Ishwaran2008].
+The expected mortality is defined by
+
+$$
+\sum_t -\log(\hatS_i(t)) = \sum_t \hatH_i(t)
+$$
+
+and is closely related to the calibration measure defined in @sec-alpha.
+This represents the expected number of events for individuals with similar characteristics to $i$.
+This is a measure of risk as a larger value indicates that among individuals with a similar profile, there is expected to be a larger number of events and therefore have greater risk than those with smaller ensemble mortality.
+For example, say for an individual, $i$, we have: $(t, \hatS_i(t)) = (0, 1), (1, 0.8), (2, 0.4), (3, 0.15)$ then, $(t, \hatH_i(t)) = (0, 0), (1, 0.10), (2, 0.40), (3, 0.82)$ and their relative risk prediction is $\sum_t \hatH_i(t) = 0 + 0.10 + 0.4 + 0.82 = 1.32$.
 
 ## Predicting Survival Times {#sec-survtsk-time}
```

Comment:
I think this paragraph needs some work. I think the essensce is that we can't predict the expectation if the estimated distribution is improper/the sample is biased towards shorter event times. But it can be done for parametric models (even with machine learning)

Reply:
and even in the presence of censoring